### PR TITLE
infra: fix bump_license_year.yml not adding year to commit message

### PR DESCRIPTION
**Pull request by Vyom-Yadav**
Source: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable

Proof of working: https://github.com/Vyom-Yadav/actions-test/commit/8ae3f4490e16f3bbba5f252f96b03e15198e8122

### CODE REVIEW

**Review by romani — APPROVED** [review]

File: .github/workflows/bump-license-year.yml [hunk 1]
```diff
@@ -7,7 +7,8 @@
 name: "Bump license year"
 on:
   schedule:
-    - cron: '0 16 1 1 *'
+    - cron: "0 0 1 1 *"
```
Comment (romani) [inline .github/workflows/bump-license-year.yml:10]: Please make hour to be 2

As. 2h after https://github.com/checkstyle/checkstyle/blob/54616f662130aa0ca5afe45f72d36cdefb78a781/.github/workflows/bump_license_year.yml#L10

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: .github/workflows/bump-license-year.yml (hunk 1 above)
Comment (romani) [reply .github/workflows/bump-license-year.yml:10]: Please update all others to be 2h after main repo

Comment (Vyom-Yadav) [reply .github/workflows/bump-license-year.yml]: Done.

**Review by romani — APPROVED** [review]

**Pull request merged**